sourabh/Html_assignment6.txt

```diff
@@ -0,0 +1,2 @@
+Q 6.1 When we link to an image from an HTML document, does the image have to be in the same directory as the HTML page or can it exist anywhere on the Internet?
+  ANS:- Not necessary to put image in same directory,Image can exist anywhere on the internet.
```

Comment:
And what happens if that image does not exist on internet? Also, did you understand the "img" tag and its attribute completely?

Reply:
if that image does not exist on the internet it will look broken or crack image and alt attribute gives name of the image.